main.py

```diff
@@ -18,14 +19,16 @@
     dect = yolo.Yolo(cfg_file, weight_file,namesfile, datafile)
     
     # Initiate tracker object
-    ot = Centroid_tracker()
-    
```

Comment:
Do not remove the code from the main untill is redundant, its better to comment it out for now.